app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,62 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  let lowerBoundError
+  let startingPointError
+  let upperBoundError
+  let startingPointNumber
+  let lowerBoundNumber
+  let upperBoundNumber
+  // if the input values aren't null the number is cast to BigNumber with 18 decimals and precision value of 2
+  if (startingPoint !== null) {
+    startingPointNumber = formatBigNumber(startingPoint, 18, 2)
```

Comment:
You can simplify this to, `startingPointNumber = startingPoint && formatBigNumber(startingPoint, 18, 2)`, same for the other values including the `!== undefined` checks

Reply:
good call